decentraland/unity-explorer
# fix: handle failed Wearable DTO

## What does this PR change?

Potentially fixes #1470 

The profile `0x08BF36607ca7912e45625dBe270D3b26604CEA1f` had an empty wearable DTO `urn:decentraland:off-chain:base-avatars:00_EmptyEarring)`. With this PR, we started handling this scenarios.

If a wearable DTO cannot be resolved, an exception is added and the avatar flow can continue normally.

## How to test the changes?


1. Launch the explorer
2. Use the debug menu to load the profile `0x08BF36607ca7912e45625dBe270D3b26604CEA1f`. The avatar should appear

![image](https://github.com/user-attachments/assets/22df1a1f-121b-4674-9755-aff5a7389b40)


## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
